core/src/main/java/com/netflix/conductor/core/utils/ParametersUtils.java

```diff
@@ -203,12 +204,18 @@ private Object replaceList(List<?> values, String taskId, DocumentContext io) {
     }
 
     private Object replaceVariables(String paramString, DocumentContext documentContext, String taskId) {
-        String[] values = paramString.split("(?=(?<!\\$)\\$\\{)|(?<=\\})");
+        String[] values = paramString.split("(?=(?<!\\$)\\$\\{)|(?<=})");
```

Comment:
Could we add a comment here for the regex?

Reply:
That would have been helpful. The original author did not include a comment, i merely removed the redundant escape character.